libkbfs/test_common.go

```diff
@@ -70,11 +70,23 @@ func setTestLogger(config Config, t logger.TestLogBackend) {
 	config.SetLoggerMaker(testLoggerMaker(t))
 }
 
+// NewConfigForTest returns a ConfigLocal object suitable for testing.
+//
+// TODO: Decouple this from NewConfigLocal.
+func NewConfigForTest() *ConfigLocal {
+	config := NewConfigLocal()
```

Comment:
`NewConfigLocal` creates a `BlockOpsStandard` with 100 workers. You need to either `Shutdown` the old one first before reassigning it, or factor out the `BlockOpsStandard` creation into a separate method that is called by both `NewConfigLocal` and `NewConfigForTest`. I'd prefer the second approach.